server/server.js

```diff
@@ -11,13 +11,24 @@ const {applyAuthMiddleware} = require('./security')
 const schemaParser = require('./lib/schemaParser')
 const schemaListenerCreator = require('./lib/schemaListeners/schemaListenerCreator')
 
-function newExpressApp (keycloakConfig, graphqlEndpoint) {
+function newExpressApp (options, middlewares) {
```

Comment:
gonna leave these names as they are as its implied in this case I think - sound ok?

Reply:
@cfoskin yep the names are fine in this context. It was just about the naming in the places where we call the function 😄